src/Microsoft.DotNet.Build.Tasks/Microsoft.DotNet.Build.Tasks.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Unintentional, thanks for catching this. Looking at the change in ASCII shows this:

    -???<?xml version="1.0" encoding="utf-8"?>
    +<?xml version="1.0" encoding="utf-8"?>

After reverting it, it comes back after adding a file in Visual Studio and saving. (But not just when building or loading.) I'm not too familiar with what tools expect which formats... do you have a suggestion?